third_party/pyth/price-service/src/ws.ts

```diff
@@ -34,7 +35,7 @@ export type ServerMessage = ServerResponse | ServerPriceUpdate;
 
 export class WebSocketAPI {
   private wsCounter: number;
-  private priceFeedClients: Map<HexString, Set<WebSocket>>;
+  private priceFeedClients: Map<HexString, [Set<WebSocket>, boolean]>;
```

Comment:
@ali-bahjati I've used a tuple instead of a Map because typescript does strict equality check on the keys

Reply:
```
Map<HexString, Map<WebSocket, boolean>> priceFeedClientVerbosity;

priceFeedClientVerbosity.get(priceFeed).get(ws) => verbosity 

if (!priceFeedClientVerbosity.has(priceFeed)) { 
 priceFeedClientVerbosity.set(priceFeed, new Map())
}

priceFeedClientVerbosity.get(priceFeed).set(ws, givenVerbosity)
```